## üóíÔ∏è Precourse Materials

Videos to watch before the course start:
- [Introduction to Galaxy](https://www.youtube.com/watch?v=64oS5uXVRV0)
- [Introduction to Genome Assembly](https://youtu.be/-EX_G1griZE?si=iQMFxHw7OJtg-VbD)
- [Introduction to NGS Mapping](https://www.youtube.com/watch?v=zuRF_uPTY-Q)
- [Introduction to InSaFlu](https://youtu.be/8AGaNrCGmtI?si=bYhk5-lFBGjaEAQF)
- [A brief history of Computional Biology and Sequencing](https://youtu.be/idl6oq-MxbM?si=A3ShRWdwoVkjgXqk&t=575)

Publications to read:
- [Twenty years of bacterial genome sequencing](https://www.nature.com/articles/nrmicro3565) - Discusses the history of sequencing technology
- [So you want to be a computational biologist?](https://www.nature.com/articles/nbt.2740) - Advice by two bioinformaticians
- [A primer on sequencing technology](https://www.nature.com/articles/nrg2626) - In depth review of sequencing technology

---
## üõ†Ô∏è Requirements - Please signup before the course!

Before the start of the course, please create an account (using your institutional email address) for the follow web services (free):

- [INSaFlu](https://insaflu.insa.pt/accounts/register/) 
- [Galaxy](https://usegalaxy.org.au/login/start) Click `Register here` button down the bottom
- [GISAID](https://gisaid.org/register/)

Please also download the following programs and install on your laptop:

- [MEGA](https://www.megasoftware.net/)

---

## Day 1 - 2024-06-03

| Time        | Activities                                            | Speaker            |
|-------------|-------------------------------------------------------|--------------------|
| 0900-0930 | Registration                                          |                    |
| 0930-1030 | Welcome Remarks from NIV and WHO CC Melbourne         |                    |
|             | Introduction of trainers and trainers                 |                    |
|             | Housekeeping                                          |                    |
|             | Pre-course quiz                                       |                    |
|             | Group photo                                           |                    |
| 1030-1100 | Morning tea                                           |                    |
| 1100-1200 | Lecture 1: eGISRS genomic surveillance of influenza and RSV | Yi-Mo Deng               |
|             | Lecture 2: Overview of wet lab training               | Yi-Mo Deng         |
|             | Lecture 3: Overview of bioinformatics traininig       | Clyde Dapat        |
| 1200-1300 | Lunch                                                 |                    |
| 1300-1400 | RT-PCR set-up for Flu A/B NGS                         |                    |
| 1400-1500 | RT-PCR set-up for RSV NGS                             |                    |
| 1500-1530 | Afternoon tea                                         |                    |
| 1600-1700 | Lecture 4: Bioinformatics for Flu and RSV NGS         | Clyde Dapat        |


## Day 2 - 2024-06-04

| Time        | Activities                                            | Speaker            |
|-------------|-------------------------------------------------------|--------------------|
| 0900-1000 | TapeStation set up for Flu RT-PCR concentration check |                    |
| 1000-1030 | Mini lecture: concentration calculation and RSV PCR products combination | Yi-Mo Deng              |
| 1030-1100 | Morning tea                                           |                    |
| 1100-1200 | TapeStation set up for RSV RT-PCR concentration check | Yi-Mo Deng         |
| 1200-1300 | Lunch                                                 |                    |
| 1300-1400 | Flu NGS: normalization and library preparation        |                    |
| 1400-1500 | RSV NGS: concentration calculation and normalization  |                    |
| 1500-1530 | Library preparation for RSV                           |                    |
| 1530-1600 | Afternoon tea                                         |                    |
| 1600-1700 | Flow cell check, combine Flu and RSV libraries, load flow cell for NGS |                    |


## Day 3 - 2024-06-05

| Time        | Activities                                            | Speaker            |
|-------------|-------------------------------------------------------|--------------------|
| 0900-0930 | Retrieve NGS data                                     |                    |
| 0930-1100 | Load Flu NGS data on computer                         |                    |
| 1100-1130 | Morning tea                                           |                    |
| 1130-1230 | Quality control with FastQC                           | Clyde Dapat        |
| 1230-1330 | Lunch                                                 |                    |
| 1330-1530 | Genome assembly - Flu                                 | Clyde Dapat        |
| 1530-1600 | Afternoon tea                                         |                    |
| 1600-1700 | Genome assembly - Flu                                 |                    |

## Day 4 - 2024-06-06

| Time        | Activities                                            | Speaker            |
|-------------|-------------------------------------------------------|--------------------|
| 0900-1000 | Load RSV data and quality control of RSV NGS data     |                    |
| 1000-1100 | Genome assembly - RSV                                 | Clyde Dapat        |
| 1100-1130 | Morning tea                                           |                    |
| 1130-1230 | Genome assembly - RSV                                 |                    |
| 1230-1330 | Lunch                                                 |                    |
| 1330-1530 | QC on consensus sequences                             | Clyde Dapat        |
| 1530-1600 | Afternoon tea                                         |                    |
| 1600-1700 | Online tools (Nextstrain, Nextclade, FluSurver)       | Clyde Dapat        |


## Day 5 - 2024-06-07

| Time        | Activities                                            | Speaker            |
|-------------|-------------------------------------------------------|--------------------|
| 0900-1100 | Phylogenetic analysis                                 | Clyde Dapat        |
| 1100-1130 | Morning tea                                           |                    |
| 1130-1230 | Phylogenetic analysis                                 |                    |
| 1230-1330 | Lunch                                                 |                    |
| 1330-1430 | Uploading sequences to GISAID                         | Clyde Dapat        |
| 1430-1500 | Open forum, Q&A                                       |                    |
| 1500-1530 | Afternoon tea                                         |                    |
| 1530-1600 | Post-course quiz and course evaluation                |                    |
| 1600-1700 | Awarding of certificates and closing remarks          |                    |

---

## Other resources:

- Benchmarking reveals superiority of deep learning variant callers on bacterial nanopore sequence data, [https://doi.org/10.1101/2024.03.15.585313](https://doi.org/10.1101/2024.03.15.585313)


